src/cost_functions.jl

```diff
@@ -10,6 +10,18 @@ function (f::Regularization)(p)
   f.λ*value(f.penalty, p)
 end
 
+function prior_loss(prior,p)
+  ll = 0.0
+  if eltype(prior) <: UnivariateDistribution
+    for i in 1:length(prior)
+      ll -= logpdf(prior[i],p[i])
+    end
+  else
+    ll -= logpdf(prior[1],p)
```

Comment:
if it's just a multivariate distribution, does it need to be indexed?

Reply:
Taking array as input to maintain uniformity?